### PR TITLE
PLANET-5747 Counter block: Counter is not inserted into EN Form

### PR DESCRIPTION
**Pull request by dpivo**
Ref: https://jira.greenpeace.org/browse/PLANET-5747

---

Moved the script that appends the counter bar to the en-form block into the Frontend file and removed the old script

### CODE REVIEW

File: assets/src/blocks/Counter/CounterFrontend.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ export class CounterFrontend extends Component {
     if (completed_api && completed_api.startsWith('https://')) {
       window.addEventListener('updateCounter', function () { counter.calculateRemaining(); }, false);
     }
+    // Append the counter inside the En-Form
+    $('.counter-style-en-forms-bar').appendTo('.enform-extra-header-placeholder').show();
```
Comment (lithrel) [inline assets/src/blocks/Counter/CounterFrontend.js:25]: As we're trying to remove jQuery from the frontend, can you do it in vanilla JS ? something like (didn't test :grimacing:)
```js
let counter_placeholder = document.querySelector('.enform-extra-header-placeholder');
let counter = document.querySelector('.counter-style-en-forms-bar');
if (counter_placeholder && counter) {
    counter_placeholder.appendChild(counter);
}
```
If you do you can close this one too https://jira.greenpeace.org/browse/PLANET-5663 :partying_face:

Comment (comzeradd) [reply assets/src/blocks/Counter/CounterFrontend.js:25]: This one I think :grin: https://jira.greenpeace.org/browse/PLANET-5766

File: assets/src/blocks/Counter/CounterFrontend.js [hunk 2]
```diff
@@ -21,6 +21,10 @@ export class CounterFrontend extends Component {
     if (completed_api && completed_api.startsWith('https://')) {
       window.addEventListener('updateCounter', function () { counter.calculateRemaining(); }, false);
     }
+    // Append the counter inside the En-Form
+    var counterBar = document.querySelector('.counter-style-en-forms-bar');
+    var enFormHeader = document.querySelector('.enform-extra-header-placeholder');
+    enFormHeader.append(counterBar);
```
Comment (lithrel) [inline assets/src/blocks/Counter/CounterFrontend.js:27]: Tested with a counter with different style, you should check for at least the existence of the counter `.counter-style-en-forms-bar` before appending it, if you don't it adds null :grin: 
Screenshot from 2020-12-10 14-32-49

**Review by lithrel — CHANGES_REQUESTED** [review]

**Review by lithrel — APPROVED** [review]
:+1:

**Comment by comzeradd** [comment]
@dpivo can you rebase with master so the tests pass?

**Pull request merged**